src/Models/ElementalArea.php

```diff
@@ -187,6 +187,11 @@ public function onBeforeWrite()
      */
     public function getOwnerPage()
     {
+        // You can't find the owner page of a area that hasn't been save yet
+        if ($this->ID == 0) {
```

Comment:
Any reason we're not using `isInDb()` here?

```suggestion
        if (!$this->isInDB()) {
```

Reply:
Because ... I wanted to know if you were paying attention :roll_eyes: